rfcs/text/0000-extend-threat-group-software.md

```diff
@@ -0,0 +1,167 @@
+# 0000: Extend Threat Fieldset
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+Currently the `threat` fieldset includes tactic, technique, and sub-techniques from the ATT&CK framework. ATT&CK also includes groups and software that we can easily add to the existing fieldset to include all of the ATT&CK framework. While these fields are directly referenced within the ATT&CK framework, they can also be used with other frameworks if `threat.framework` expands to use more than ATT&CK.
```

Comment:
Thanks for including the consideration for other frameworks. `threat.*` is certainly very much aligned with ATT&CK today, but we don't want it to come off as _exclusively_ ATT&CK focused.

Reply:
Agreed, ATT&CK may currently be the contemporary favorite but we should be mindful that a) it isn't universally relied upon and b) it's not permanent.